src/components/home/Home.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import SearchBar from 'components/searchBar';
+import GifList from 'components/gifList';
+import trendingActions from 'actions/trending';
+import searchActions from 'actions/search';
+import { StyledHeading1 } from '../Heading1.style'
+import { EmptyPlaceholder } from '../EmptyPlaceholder.style'
+
+export class Home extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchValue: ''
+    }
+  }
+
+  componentDidMount = () => {
+    this.props.getTrendingGifs();
+  }
+
+  handleSearch = (e) => {
+    let value = e.target.value;
+    this.setState({
+      searchValue: value
+    })
+    this.props.getSearchGifs(value);
+  }
+
+  render() {
+    let searchToggle = null;
+
+    if(!this.props.trending) return null
```

Comment:
This code is basically the same than before, therefore why could be useful to have this logic in the Redux store.